libraries/AP_Follow/AP_Follow.cpp

```diff
@@ -320,7 +322,75 @@ void AP_Follow::handle_msg(const mavlink_message_t &msg)
             _sysid.set(msg.sysid);
             _automatic_sysid = true;
         }
+        updated = true;
+        break;
+    }
+    case MAVLINK_MSG_ID_FOLLOW_TARGET: {
+        // get estimated location and velocity (for logging)
```

Comment:
I think we can reduce some duplicate code by moving these lines lower down, outside the select/case because the results are only used for logging.

Reply:
they are only used in logging, so I moved them down to the logging code